tests/mesh/boundary_info.C

```diff
@@ -20,41 +21,35 @@ public:
   CPPUNIT_TEST_SUITE( BoundaryInfoTest );
 
   CPPUNIT_TEST( testMesh );
+  CPPUNIT_TEST( testEdgeBoundaryConditions );
 
   CPPUNIT_TEST_SUITE_END();
 
 protected:
 
-  Mesh* _mesh;
-
-  void build_mesh()
-  {
-    _mesh = new Mesh(*TestCommWorld);
-
-    MeshTools::Generation::build_square(*_mesh,
-                                        2, 2,
-                                        0., 1.,
-                                        0., 1.,
-                                        QUAD4);
-  }
-
 public:
   void setUp()
   {
-    this->build_mesh();
```

Comment:
Recall too that everything is wiped between tests, so if you want a mesh for a test, you need it in `setUp()` (and the corresponding cleanup in `tearDown()`.